src/control/cmd/daos_server/auto.go

```diff
@@ -24,6 +24,9 @@ import (
 	"github.com/daos-stack/daos/src/control/server/storage"
 )
 
+var ErrTmpfsNoExtMDPath = errors.New("--use-tmpfs-scm will generate an md-on-ssd config and so " +
```

Comment:
`--use-tmpfs-scm` seems a little bit confusing for me as it should also be used for generating configuration without Block Device tiers (only SCM storage tiers).

FYI, during my last experiment, I have also noticed that the `control_metadata.path` could be used with SCM storage tiers only. Not sure, if such configuration should be allowed.

Reply:
`--use-tmpfs-scm` is probably not the best name now MD-on-SSD exists but I think it's okay for now. `--scm-only` allows for non-NVMe se!ups.

https://daosio.atlassian.net/browse/DAOS-13519 now addresses your second point

Reply:
Fair enough and thanks for the second point PR.